### ISSUE TITLE
question about the busco version in funannotate

### ISSUE THREAD

**Issue by dingxm** [nextgenusfs/funannotate#1046]
Hi ,
I have  plant genome which is around 1.5Gb. I do not have RNAseq data. I tried the funannotate for the gene prediction. It did a pretty good job. I have checked the busco score of the genome which is around 98% complete using Busco V5.7. When I run the  funannotate, it showed the busco score is around 87% at the genome level. Is this because the version of busco in funannotate is V2 not V5?  The busco score for the protein predicted by funannotate is around 82% even I used Busco V5.7. Is that because when funannotate perform the gene prediction only used 87% genes not all the busco genes for the augustus training.  If we used all the busco genes for the augustus training, will the busco score for the predicted protein got improved?
Thanks

**Comment by edwhisnant** [comment]
I am wondering about a similar issue, as I preliminarily screened my genomes using the most recent version of BUSCO (5.7), but funannotate is running v2 and is using an outdated database, leading to lower BUSCO completeness scores.  You may want to manually install the updated databases and just run using the preinstalled BUSCO version.

This was partially answered [here](https://github.com/nextgenusfs/funannotate/issues/637).

**Comment by nextgenusfs** [comment]
Yes you can download the v10 models and use them as I indicated in that other post. I've since rewritten the BUSCO tool as buscolite https://github.com/nextgenusfs/buscolite. You can read about why and that the metaeuk results aren't meaningful for gene predictions there. 

Funannotate2 is nearly ready (probably release sometime this week) which should solve these issues. After f2 release all dev on this repo (at least by me) will stop.

**Comment by BWANG2016** [comment]
I am encountering the same issue: the latest BUSCO db12 no longer includes the lengths_cutoff file, but funannotate predict uses an older version of BUSCO that still requires it.
